openshift/templates/dockerhub-imagestreams/nginx.yaml

```diff
@@ -47,7 +47,7 @@ objects:
         source: dockerhub
     spec:
       tags:
-        - name: "1.21.6-alpine"
+        - name: "1.23.1-alpine"
           from:
             kind: DockerImage
             name: "docker-remote.artifacts.developer.gov.bc.ca/nginx:1.21.6-alpine"
```

Comment:
This needs to 1.23.1 as well, as this is what pulls the correct image.

Reply:
d'oh. Thanks for the catch.